packages/ui-test-sandbox/src/utils/reactComponentWrapper.tsx

```diff
@@ -146,18 +146,17 @@ class ReactComponentWrapper {
   }
 
   unmount() {
-    return new Promise((resolve, reject) => {
```

Comment:
I dont knpow why this was a Promise, there is no async code here

Reply:
I guess this is some kind of hacky solution to wait 1 tick of the event loop. Kinda the same if you were to do a `setTimeout(.., 0)` call.